plugins/lua/stockpiles.lua

```diff
@@ -428,10 +428,10 @@ end
 StockpilesOverlay = defclass(StockpilesOverlay, overlay.OverlayWidget)
 StockpilesOverlay.ATTRS{
     desc='Shows a panel when a stockpile is selected for stockpile automation.',
-    default_pos={x=24, y=-6},
+    default_pos={x=5, y=44},
     default_enabled=true,
-    viewscreens='dwarfmode/Stockpile/Some',
-    frame={w=65, h=4},
+    viewscreens='dwarfmode/Stockpile/Some', -- hmm, how do I close it when 'dwarfmode/CustomStockpile' is also up?
```

Comment:
I don't know why we have it this way right now. let me look into it. Maybe we can get rid of `dwarfmode/CustomStockpile` as a focus string and instead add a suffix to `dwarfmode/Stockpile/Some`. E.g. main map stockpile selected would be `dwarfmode/Stockpile/Some/Default` and customization screen would be `dwarfmode/Stockpile/Some/Customize`. Same goes for `dwarfmode/StockpileLink` and `dwarfmode/StockpileTools`.

Reply:
Fixed in https://github.com/DFHack/dfhack/pull/4612
You can now use `dwarfmode/Stockpile/Some/Default`

Reply:
This seems to work without changing it from `dwarfmode/Stockpile/Some` - which is great! - I'm just not sure I understand what `dwarfmode/Stockpile/Some/Default` represents.